rosidl_generator_py/resource/msg.py.template

```diff
@@ -0,0 +1,27 @@
+# Generated by rosidl_generator_py
+
+class @(spec.base_type.type):
+@[if len(spec.fields) > 0]@
+    __slots__ = ['@("', '".join([field.name for field in spec.fields]))']
+    def __init__(self, **kwargs):
+@[for field in spec.fields]@
+@[if field.default_value]@
+        self.@(field.name) = kwargs.get(
+            '@(field.name)', @value_to_py(field.type, field.default_value))
+@[else]@
+        self.@(field.name) = kwargs.get('@(field.name)')
```

Comment:
Doesn't this fail for all fields without a default value if the argument is not passed to the constructor?